src/components/ha-code-editor.ts

```diff
@@ -32,6 +32,8 @@ export class HaCodeEditor extends ReactiveElement {
 
   @property() public mode = "yaml";
 
+  @property() public name;
+
```

Comment:
This shouldn't be needed

Reply:
It is :)

Trigger event changed assume `FormElement` (from `mwc-base`, which our code editor isn't).
This we need to supply the bare minimum we depend upon, which is the name (used by the `handleChangeEvent` in the `ha-automation-trigger-row.ts` file.

Reply:
You can have the name on the element in `ha-automation-trigger-row.ts` without needing to add the property in the element. It will still set the name attribute and should be gettable

Reply:
Ok, so I'm a noob at frontend stuff and you would need to help / provide a hint me on this one.

It tried setting the name on HTML, but apparently: that is not enough to access it as a property. So, I guess I'm missing something?

Reply:
Use can use `.name=${"name"}`

This will set it as a property. You are correct that it needs the Attr when setting it with just `name=""`